RadixWallet/Core/DesignSystem/Components/AppTextField.swift

```diff
@@ -169,7 +173,7 @@ public struct AppTextField<FocusValue: Hashable, Accessory: View, InnerAccessory
 				.cornerRadius(.small2)
 				.overlay(
 					RoundedRectangle(cornerRadius: .small2)
-						.stroke(accentColor(border: !isEnabled), lineWidth: 1)
+						.stroke(accentColor(border: !isFocused), lineWidth: 1)
```

Comment:
I think we can reverse the condition for the `border` as we display a darker one when the field is focused: 
```Swift
.stroke(accentColor(border: isFocused), lineWidth: 1)
```
```Swift
	private func accentColor(border: Bool) -> Color {
		switch hint?.viewState.kind {
		case .none:
                     border ? .app.gray1 : .app.gray4
		case .info:
			.app.gray1
		case .error:
			.app.red1
		case .warning:
			.app.alert
		}
	}
```